File: src/S1/BOofOne/Python/student_helper.py
# student_helper.py

def show_name_at(students, index):
    if 0 <= index < len(students):
        print(f"Student: {students[index]} at index {index}")
    else:
        print(f"Error: Index {index} is out of bounds of total {
              len(students)}.")

File: src/S1/BOofOne/Go/studentHelper.go
// studentHelper.go

package main

import "fmt"

func showNameAt(students []string, index int) {
	if index >= 0 && index < len(students) {
		fmt.Printf("Student: %s at index %d\n", students[index], index)
	} else {
		fmt.Printf("Error: Index %d is out of bounds of total %d.\n", index, len(students))
	}
}
